docs/mods.md

```diff
@@ -0,0 +1,72 @@
+# Mods
+
+This is a list of known mods to DevilutionX. Feel free to extend it by sending a PR.
+
+## Infernity
```

Comment:
Infernity is technically a Devilution mod and not DevilutionX, and as it's a rather old base it's probably hard to pick pices from it. Still worth having on the list.

Reply:
Didn't know that. @qndel, rebase! :)

Reply:
He is waiting for me to rewrite all the code first :D

Reply:
lol

Reply:
infernity's biggest feature were custom resolutions :) everything else is minor compared to that and I could port it to devilutionx in a day